templates/output.html.twig

```diff
@@ -31,7 +31,12 @@
                 </div>
             </div>
             <div class="form-group">
-                <label for="lang">{{ msg('language-code') }}</label>
+                <label id="optional-lang-label" for="lang" class="lang-label {% if engine == 'transkribus' %}hidden{% endif %}">
+                    {{ msg('language-code') }}
+                </label>
+                <label id="transkribus-lang-label" for="lang" class="lang-label {% if engine != 'transkribus' %}hidden{% endif %}">
+                    {{ msg('transkribus-language-code') }}
+                </label>
```

Comment:
The #lang select below doesn't get `required` on first load (i.e. when `?engine=transkribus` is given in the URL).